src/components/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import { Link } from 'react-router-dom';
+
+import { useHeader } from 'src/components/Header/hook';
+
+import { BurgerIcon } from 'src/components/_common/icons/BurgerIcon';
+import { CloseIcon } from 'src/components/_common/icons/CloseIcon';
+import { SwitchThemeIcon } from 'src/components/_common/icons/SwitchThemeIcon';
+import { LightDarkTheme } from 'src/components/LightDarkTheme/LightDarkTheme';
+
+import { MENU_PATH } from 'src/features/Router/path';
+import { PLANET_OBSERVER_PATH } from 'src/features/Router/path';
+
+import './header.css';
+
+
+type HeaderProps = {
+  showThemeToggle: boolean;
+  showBurgerIcon: boolean;
```

Comment:
i would recommend to make both as not required and rename "showBurger" with "isMenuActive" to set it only for Menu page.

Reply:
Done